### ISSUE TITLE
Adding Opensnowmap

### ISSUE THREAD

**Issue by stell** [leaflet-extras/leaflet-providers#376]
Hi there,
opensnowmap has some nice information about skiing, slopes and lifts. Would it be possible to add that map and overlay to leaflet-providers?

Here an example view:
http://www.opensnowmap.org/?zoom=12&lat=45.33404&lon=6.59113&layers=snowbase&marker=false&marker=true

**Comment by brunob** [comment]
Nice, i can read this from their website :

```
Tiles from Opensnowmap.org comes in two flavours: one with the relief (hillshade and contour lines), and one with ski pistes only. You are welcome to use the pistes-only tiles (without relief) on your website, but it is madantory to :

    use a valid referer
    prohibit bulk download from your app
    credit Data (c) www.openstreetmap.org & contributors ODBL and www.opensnowmap.org CC-BY-SA.

The base topo map or the piste+relief overlay are too demanding in server ressources to be shared easily.
```

Would you mind to provide a PR for that ?

**Comment by stell** [comment]
PR provided

**Comment by brunob** [comment]
Fixed by #379 thx again :)
